backend/ipmi.pm

```diff
@@ -95,6 +95,7 @@ sub restart_host {
         last if $ret =~ m/is on/;
         $self->ipmitool('chassis power on');
     }
+    1;
```

Comment:
Is this just a leftover?

Reply:
Actually I thought every good perl module should end with a success status but I could remove that without a failing test so I keep that change out of this PR. No problem.

Reply:
But this is the middle of a function, not the end of the module :-)

Reply:
oh, wow. Well I have removed it because I misread your comment. That was stupid of me.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/1587

By the way, because obviously we should not be blocked by hours-long waiting time for travis CI I continued with https://github.com/os-autoinst/os-autoinst/pull/1576